src/main/java/seedu/address/ui/PersonCard.java

```diff
@@ -76,6 +91,40 @@ public PersonCard(Person person, int displayedIndex) {
                 .forEach(clientSource -> clientSources.getChildren().add(
                         new Label(clientSource.clientSourceName)
                 ));
+
+        setPriorityShape(person);
+    }
+
+    private void setPriorityShape(Person person) {
+        switch (person.getPriority().value) {
+        case "low":
+            priorityShape.setBorder(new Border(new BorderStroke(ColorPicker.TURQUOISE_BORDER,
+                    BorderStrokeStyle.SOLID, CornerRadii.EMPTY, new BorderWidths(BORDER_SIZE))));
+            priorityShape.setBackground(new Background(new BackgroundFill(ColorPicker.TURQUOISE, CornerRadii.EMPTY,
+                    Insets.EMPTY)));
+            break;
+        case "medium":
+            priorityShape.setBorder(new Border(new BorderStroke(ColorPicker.ORANGE_BORDER,
+                    BorderStrokeStyle.SOLID, CornerRadii.EMPTY, new BorderWidths(BORDER_SIZE))));
+            priorityShape.setBackground(new Background(new BackgroundFill(ColorPicker.ORANGE, CornerRadii.EMPTY,
+                    Insets.EMPTY)));
+            break;
+        case "high":
+            priorityShape.setBorder(new Border(new BorderStroke(ColorPicker.RED_BORDER,
+                    BorderStrokeStyle.SOLID, CornerRadii.EMPTY, new BorderWidths(BORDER_SIZE))));
+            priorityShape.setBackground(new Background(new BackgroundFill(ColorPicker.RED, CornerRadii.EMPTY,
+                    Insets.EMPTY)));
+            break;
+        case "undefined":
+            priorityShape.setBorder(new Border(new BorderStroke(ColorPicker.WHITE_BORDER,
+                    BorderStrokeStyle.SOLID, CornerRadii.EMPTY, new BorderWidths(BORDER_SIZE))));
+            priorityShape.setBackground(new Background(new BackgroundFill(Color.WHITE, CornerRadii.EMPTY,
```

Comment:
Is there a difference between ColorPicker.WHITE_BORDER and Color.WHITE?

Reply:
Yes, there is, white border is slightly darker